joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 50

joint0_position_controller:
  type: effort_controllers/JointPositionController
  joint: arm_joint_0
  pid: {p: 4.0, i: 1.0, d: 0.0}

joint1_position_controller:
  type: effort_controllers/JointPositionController
  joint: arm_joint_1
  pid: {p: 4.0, i: 1.0, d: 0.0}

camera_position_controller:
  type: effort_controllers/JointPositionController
  joint: camera_joint
  pid: {p: 4.0, i: 1.0, d: 0.0}
